pcsx2/USB/usb-pad/usb-turntable.cpp

```diff
@@ -466,9 +466,9 @@ namespace usb_pad
 	gsl::span<const SettingInfo> DJTurntableDevice::Settings(u32 subtype) const
 	{
 		static constexpr const SettingInfo info[] = {
-			{SettingInfo::Type::Integer, "TurntableMultiplier", TRANSLATE_NOOP("USB", "Turntable Multiplier"),
-				TRANSLATE_NOOP("USB", "Multiplies the turntable rotation speed by a constant. Useful for using Xbox 360 turntables."),
-				"1", "1", "100", "1", "%d", nullptr, nullptr, 1.0f}};
+			{SettingInfo::Type::Float, "TurntableMultiplier", TRANSLATE_NOOP("USB", "Turntable Multiplier"),
+				TRANSLATE_NOOP("USB", "Apply a multiplier to the turntable"),
+				"0.00", "0.00", "100.0", "1.0", "%d", nullptr, nullptr, 1.0f}};
```

Comment:
The `%d` here should probably be `%.0fx` or something along those lines (to display as 1x).

Otherwise LGTM.

Reply:
Oops yeah, i must have hit ctrl+z a few too many times haha